.github/kind.yml

```diff
@@ -0,0 +1,13 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  image: "kindest/node:v1.20.2"
+- role: control-plane
+  image: "kindest/node:v1.20.2"
+- role: control-plane
+  image: "kindest/node:v1.20.2"
+- role: worker
+  image: "kindest/node:v1.20.2"
+- role: worker
+  image: "kindest/node:v1.20.2"
```

Comment:
Is there a reason to have more than 1 CP node for tests?
Also, can it be a matrix of versions, so that we could cover different supported k8s versions (e.g. latest 1.16-1.20)?

Reply:
I wanted to replicate the testing as much close to production as possible so that's why i chose to run in HA for CP.

Reply:
Wont one `control-plane` and  two `worker` node satisfy minimum production  setting?

Reply:
i don't think that's HA or really production level. But running with 5 node setup there seems to be a [bug](https://github.com/kubernetes-sigs/kind/issues/1288) in KinD so will use a 3 node setup.